lldb/source/Plugins/SymbolFile/NativePDB/SymbolFileNativePDB.cpp

```diff
@@ -618,18 +618,14 @@ lldb::TypeSP SymbolFileNativePDB::CreateSimpleType(TypeIndex ti,
 }
 
 static std::string GetUnqualifiedTypeName(const TagRecord &record) {
-  if (!record.hasUniqueName()) {
-    MSVCUndecoratedNameParser parser(record.Name);
-    llvm::ArrayRef<MSVCUndecoratedNameSpecifier> specs = parser.GetSpecifiers();
-
-    return std::string(specs.back().GetBaseName());
-  }
+  if (!record.hasUniqueName())
+    return std::string(MSVCUndecoratedNameParser::DropScope(record.Name));
```

Comment:
Was this change required? Or a drive-by cleanup?

Reply:
No, this isn't required. This was basically the implementation of [`DropScope`](https://github.com/llvm/llvm-project/blob/d4f51bd53ed583cb4c8b149d5e3d31ac5f64d82a/lldb/source/Plugins/Language/CPlusPlus/MSVCUndecoratedNameParser.cpp#L99-L104) before. I changed it for clarity - to be identical with the other case.